SonarQube.Common/Resources.resx

```diff
@@ -254,4 +254,7 @@
   <data name="ERROR_FailedParsePropertiesEnvVar" xml:space="preserve">
     <value>Failed to parse properties from the environment variable '{0}'</value>
   </data>
+  <data name="ERROR_InvalidPropertyName" xml:space="preserve">
+    <value>At least one of the property name is malformed. Please check that the settings file is valid.</value>
```

Comment:
Minor: "one of the property **names** is malformed", or "At least one property name is malformed".
Also, it might be better to say "missing" in this case.